src/components/RootApp/ScalprumRoot.tsx

```diff
@@ -170,15 +170,16 @@ const ScalprumRoot = memo(
                   loadScripts: [],
                 };
               }
-              return {
+              const newManifest = {
                 ...manifest,
                 // Compatibility required for bot pure SDK plugins, HCC plugins and sdk v1/v2 plugins until all are on the same system.
-                baseURL: '/',
+                baseURL: manifest.name.includes('hac-') && !manifest.baseURL ? `${isBeta() ? '/beta' : ''}/api/plugins/${manifest.name}/` : '/',
```

Comment:
Obviously not an ideal solution here... Hopefully we can remove this in the near future when we migrate hac-dev

Reply:
They will have to update the build SDK to at least v3 for use to be able to remove it.